src/backend/cc.h

```diff
@@ -195,6 +195,7 @@ struct Srcpos
     struct Sfile **Sfilptr;     // file
     #define srcpos_sfile(p)     (**(p).Sfilptr)
     #define srcpos_name(p)      (srcpos_sfile(p).SFname)
+    short Sfilnum;              // file number
```

Comment:
So the old layout was: `unsigned`, `unsigned`, pointer, pointer, `short`. The new layout is: `unsigned`, `unsigned`, `short`, pointer, pointer. Where is the improvement?